entrypoint.sh

```diff
@@ -23,7 +23,7 @@ install_nf_core_tools(){
 }
 
 nf-core-lint() {
-    if [ -z "$INPUT_NFCORE_VERSION" ]
+    if [ "$MODE" == "release" ]
```

Comment:
```suggestion
    if [ "$INPUT_MODE" == "release" ]
```
Env variables in GH actions are prefaced with `INPUT_`